src/posix.mak

```diff
@@ -488,13 +513,13 @@ zip:
 
 #############################
 
-ifneq ($(DOCDIR),)
+#ifneq ($(DOCDIR),)
```

Comment:
Why this change, BTW?

I guess the `ifneq` was supposed to protect accidentally writing to the filesystem root... but if you think the change is not needed, I'd say to just remove it. Though, this change looks like it might have been accidentally committed.